src/main/java/com/WeatherIT/dto/LocationDTO.java

```diff
@@ -0,0 +1,28 @@
+package com.WeatherIT.dto;
+
```

Comment:
Added LocationDTO package to start the transition from javascript to java this can also be used to store location data at a later point if needed